docs/team/kidsnd274.md

```diff
@@ -4,7 +4,7 @@ title: Samuel Tee's Project Portfolio Page
 ---
 ## Project: AddressBook Level 3
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
I think this should be kept the same as well
```AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.```

Reply:
I think you need to update this, yeah? Since we are doing TeachWhat and not the addressbook level 3 anymore, but I've changed it.